### PR TITLE
update gulpfiles

### PR DESCRIPTION
**Pull request by ciss1995**

### CODE REVIEW

File: components/d2l-enrollment-card/build/lang/de.js [hunk 1]
```diff
@@ -1,27 +1,32 @@
...
-		"inactive": "Inaktiv",
-		"new": "Neu",
-		"overdue": "Überfällig",
-		"pin": "Anheften",
-		"pinActionResult": "{course} wurde angeheftet",
-		"unpin": "Lösen",
-		"unpinActionResult": "{course} wurde gelöst"
-}
-};
+<script>
```
Comment (m6jones) [inline components/d2l-enrollment-card/build/lang/de.js]:
```suggestion
<script>
```

Comment (m6jones) [reply components/d2l-enrollment-card/build/lang/de.js]: Sorry, only had a mouse. So you will need to fix the template to not include these in the build files.

Comment (ciss1995) [reply components/d2l-enrollment-card/build/lang/de.js]: Oh, I did not even notice that. Thanks

**Comment by svanherk** [comment]
I'm missing the context around this, why did this need updating?

**Comment by ciss1995** [comment]
> I'm missing the context around this, why did this need updating?

@edwardg8 told me yesterday that @m6jones noticed that those were out of date.

**Comment by m6jones** [comment]
Basically, the language files were made in hybrid and the script pushed them to work with polymer 3. Now that we don't have hybrid the build process for building these language files broke.

**Comment by edwardg8** [comment]
haha thanks Maya for finding out the problem and Cheick is actively looking for new things to work with and volunteered doing this while I was busy with enrollment card. I'm the telephone here. :)

**Pull request merged**